content/docs/services/cdn-route.md

```diff
@@ -88,7 +88,10 @@ The maximum number of domains that can be associated with a single cdn-route ser
 
 ### How to set up DNS
 
-**Note:** If you are creating a new site on cloud.gov or you are migrating an existing site to cloud.gov that can tolerate a small amount of downtime during the migration, you can skip the first step and proceed directly to [Create CNAME record(s)](#step-2-create-cname-record-s)
+**Note:** Due to changes in how CloudFront processes requests, we currently only offer certificate provisioning via DNS challenges. This means that users can no longer skip step 1 below.
+We are investigating ways to bring this feature back. 
```

Comment:
Can you add something like this:

> If you have a use case where DNS configurations are not available for you, please reach out to us at support@cloud.gov.

It's a good opportunity for us to find out how many users really use the HTTP-01 challenges.

Reply:
updated.